### PR TITLE
SortIt: Fix box momentum on drop

### PR DESCRIPTION
**Pull request by Numenter**
# Description
This will fix the buggy boxes in SortIt. Not clean but I think it works.
@RedstoneMedia would be great if you could test and confirm it.

Related to #84

## Type of change
- [x] Bug fix (non breaking change that fixes an issue)

### CODE REVIEW

**Comment by RedstoneMedia** [comment]
Tested in Godot version 3.5.0: Forcing the boxes to go to sleep seems to work 👍
I do have some small additions though: https://github.com/Numenter/Suffragium/pull/6

File: game/games/sortit/box.gd [hunk 1]
```diff
@@ -14,6 +14,21 @@ export(float) var number_scale = 0.8
...
 var number: int = 0
 var is_black: bool = false
+var force_sleep: bool = false
+
+
+func _process(_delta):
+	# Handle boxes falling through the ground (Should not happen)
+	if translation.y < -10:
+		push_warning("SortIt: Box dropped through the ground")
+		_on_despawn_timer_timeout()
```
Comment (MarioLiebisch) [inline game/games/sortit/box.gd]: Here, move the logic inside `_on_despawn_timer_timeout()` into its own function, like `despawn()` and call that instead (same inside the signal callback). I'd consider it bad practice to call such callback functions by hand, although it's certainly vlaid.

File: game/games/sortit/box.gd [hunk 2]
```diff
@@ -14,6 +14,21 @@ export(float) var number_scale = 0.8
...
+
+	if force_sleep:
+		sleeping = true
+		$CollisionShape.disabled = true
+	else:
+		sleeping = false
+		$CollisionShape.disabled = false
```
Comment (MarioLiebisch) [inline game/games/sortit/box.gd]: How about simplifying this?

```gdscript
sleeping = force_sleep
$CollisionShape.disabled = force_sleep
```
Also might be a good idea to cache `$CollisionShape` in an `onready var`?

**Review by MarioLiebisch — APPROVED** [review]
Could merge it already, but how about these two simplifications to make it a bit more readable/easier to understand?

File: game/games/sortit/box.gd (hunk 1 above)
Comment (Numenter) [reply game/games/sortit/box.gd]: I agree.

File: game/games/sortit/box.gd (hunk 2 above)
Comment (Numenter) [reply game/games/sortit/box.gd]: I changed the force_sleep logic.

**Pull request merged**